docker-compose.yml

```diff
@@ -6,6 +6,11 @@ services:
     ports:
       - "5672:5672"
       - "15672:15672"
+  rabbitmq-second:
```

Comment:
why did you added this? @rpliva

Reply:
It was probably the rest of the past. I removed it.